tests/testdata/mybuns/porter.yaml

```diff
@@ -19,6 +19,10 @@ parameters:
     minimum: 1
     maximum: 11
     default: 5
+  - name: password
```

Comment:
It would help to define a default value for this new parameter. That way existing tests that were not testing scenarios related to secrets do not need to be changed (like the uninstall integration test). This will make it easier to maintain these tests over time as we need more test cases to this bundle.

Reply:
It definitely made smoke test cleaner!